store/actions/crypto.actions.tsx

```diff
@@ -20,31 +20,43 @@ export const readData = () => {
                 }
             })
             .catch(err => { console.error(err) })
-
     }
 }
 
 export const addCripto = (textInput: string) => {
     return async (dispatch: Dispatch<Action>, getState: () => RootState) => {
+
         const { cryptoList } = getState().cripto
+        const response = await fetch(`https://data.messari.io/api/v1/assets/${textInput.toLowerCase()}/metrics?fields=id,symbol,name,market_data/price_usd,market_data/percent_change_usd_last_24_hours`)
```

Comment:
[CRITICAL] Host url should never be hardcoded in the app. Use environment variables instead

Reply:
[minor] Api requests should be wrapped in try-catch blocks